media-libs/ptex/files/ptex-2.1.28-20170213.patch

```diff
@@ -0,0 +1,173 @@
+diff -purN a/CMakeLists.txt b/CMakeLists.txt
+--- a/CMakeLists.txt	2016-04-04 18:14:05.000000000 +0100
++++ b/CMakeLists.txt	2017-02-13 22:59:10.000000000 +0000
+@@ -16,7 +16,7 @@ else ()
+ endif ()
+ 
+ if (NOT WIN32)
+-    set(CMAKE_CXX_FLAGS "-std=c++98 -Wall -Wextra -pedantic")
++    set(CMAKE_CXX_FLAGS "-std=c++98 -Wall -Wextra -pedantic ${CMAKE_CXX_FLAGS}")
+ endif ()
+ 
+ if(MSVC)
+diff -purN a/Makefile b/Makefile
+--- a/Makefile	2016-04-04 18:14:05.000000000 +0100
++++ b/Makefile	2017-02-13 22:59:10.000000000 +0000
+@@ -17,6 +17,10 @@ ifdef PRMAN_15_COMPATIBLE_PTEX
+     CMAKE_FLAGS += -DPRMAN_15_COMPATIBLE_PTEX:BOOL=TRUE
+ endif
+ 
++ifdef TOOLCHAIN
++    CMAKE_FLAGS += -DCMAKE_TOOLCHAIN_FILE=$(TOOLCHAIN)
++endif
++
+ # make V=1 shortcut for VERBOSE=1
+ ifdef V
+     VERBOSE=1
+diff -purN a/Ptex.spec.in b/Ptex.spec.in
+--- a/Ptex.spec.in	1970-01-01 01:00:00.000000000 +0100
++++ b/Ptex.spec.in	2017-02-13 22:59:10.000000000 +0000
```

Comment:
This is all unmodified upstream. I can remove the spec file section, but then it wouldn't be upstream, but ah well. Apparently it's 1970 at upstream. haha